src/Controller/Backend/BackendBase.php

```diff
@@ -71,6 +71,7 @@ public function before(Request $request, Application $app, $roleRoute = null)
         // Sanity checks for doubles in in contenttypes. This has to be done
         // here, because the 'translator' classes need to be initialised.
         $app['config']->checkConfig();
+        $app['config']->checkTaxonomy();
```

Comment:
Well, where is the right place? I didn't really know where to put it, so I followed the checkConfig way of doing things, and assumed that following established code was the way to go until we actually get to the config refactor. That way it's more predictable, no?